bigtable/scheduled-backups/src/main/java/com/example/cloud/bigtable/scheduledbackups/CreateBackupREST.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloud.bigtable.scheduledbackups;
+
+import com.example.cloud.bigtable.scheduledbackups.resource.CreateBackupMessage;
+import com.example.cloud.bigtable.scheduledbackups.resource.CreateBackupRequestHTTP;
+import com.example.cloud.bigtable.scheduledbackups.resource.PubSubMessage;
+import com.google.cloud.functions.BackgroundFunction;
+import com.google.cloud.functions.Context;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Base64;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.codehaus.jackson.map.ObjectMapper;
+
+//CHECKSTYLE OFF: AbbreviationAsWordInName
+public class CreateBackupREST implements BackgroundFunction<PubSubMessage> {
```

Comment:
Is there a reason we're including an example that uses the REST API? Are there use cases where you wouldn't be able to use the Bigtable Java Client, which I assume is the recommended method. 

tagging @kolea2 for Bigtable context

Reply:
+1 I'm curious about this too

Reply:
Many customers use REST API instead of Java Client, this is to provide an option for those.

Reply:
I think it can be confusing to have multiple similar examples. Someone using this would already be writing code in Java, so it would just add more confusion rather than be helpful. In addition, in our documentation we don't provide examples showing the REST API we strongly recommend using the client libraries over it instead. Let me know if you want to chat about this further